src/doc/gui_manual/Operators/OperatorTypes/PICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,298 @@
+
+Uniform
+    Sample uniformly along the boundary or the interior. 
+
+    * Samples in X - (Default 1) Create N samples along the X axis.
+    * Samples in Y - (Default 1) Create N samples along the Y axis.
+    * Samples in Z - (Default 1) Create N samples along the Z axis. 
+   
+Random
+    Sample randomly along the boundary or the interior. 
+
+    * Random number of samples (Default 1).
+    * Random number Seed - (Default 0) ensures that the same set of random samples is taken each time the plot is regenerated. 
+
+Boundary vs Interior Samples
+""""""""""""""""""""""""""""
+
+Samples from a circle, plane, sphere, or box can be either taken from the boundary, or the interior. For example, in the case of a plane sampling source, the samples can either lie along the edges of the planar region, or within the bounded rectangle, as shown below.
+    
+Field
+"""""
+
+See Field attributes that are common to all PICS operators.
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for numerical integrators. In addition to the Integration attributes common to all PICS operators, the Integral Curve operator supports the following attributes.
+
+Integration Direction
+"""""""""""""""""""""
+
+Sets the integration direction through time. Options are:
+    
+Forward
+    Integrate forward in time.
+
+Backward
+    Integrate backward in time.
+
+Both
+    Integrate both forward and backward in time, producing two integral curves.
+
+Forward Directionless
+    Integrate forward in time assuming a directionless vector field.
+
+Backward Directionless
+    Integrate backward in time assuming a directionless vector field.
+
+Both Directionless
+    Integrate both forward and backward in time assuming a directionless vector field, producing two integral curves.
+
+Eigen vectors are an example of a directionless vector field. To integrate using a directionless field requires that any orientation discontinuity be corrected prior to linear interpolation. That is, all vectors must be rotated to match the orientation of the trajectory. The PICS code will do this processing for standard fields (e.g non-higher order elements).
+    
+Integrator
+""""""""""
+
+See Integrator attributes that are common to all PICS operators.
+    
+Step Length
+"""""""""""
+
+See Step Length attributes that are common to all PICS operators.
+    
+Tolerances
+""""""""""
+
+See Tolerance attributes that are common to all PICS operators.
+
+Termination
+"""""""""""
+
+Integral curve termination can be controlled in several different ways. The termination is based on the most conservative criteria, so only one criteria must be met for termination. The options are:
+
+Maximum number of steps
+    The maximum number of integration steps that will be allowed. 
+
+
+Appearance
+^^^^^^^^^^
+
+The appearance tab specifies how the integral curve will be drawn. In addition to the Appearance attributes common to all PICS operators, the Integral Curve operator supports the following attributes.
```

Comment:
I wonder about *draw* versus *render*. I'm included towards *render*. But, maybe that term is to specific to *graphics* people.